src/fundamentals-2.md

```diff
@@ -81,12 +102,19 @@ Build a so called 'call to action', consisting of a image, text and button. The
 - [BEM's own reference](http://getbem.com/naming/) or [CSS-Tricks](https://css-tricks.com/bem-101/)
 
 ### Questions
-
-- What is BEM and what are its benefits?
-- What is a typical use case for a modifier?
-- What are the benefits of SASS compared to plain CSS?
-- Can you use unmodified SASS code in web browsers?
-- What is a mixin?
+<questions-component>
+  <question-component>What is BEM and what are its benefits?</question-component>
+  <answer-component>BEM stands for Block, Element, Modifier. This methology seperates styles in Blocks, which are standalone entities, Elements, which are just a part of a block and modifiers, which are used to flag Blocks or Elements. This clear structure helps to build clearly understandable CSS/Sass code.</answer-component>
+  <question-component>What are the benefits of sass compared to plain CSS?</question-component>
+  <answer-component>
+  In sass you, can declare variables, mixins, functions and baseclasses which you then can use all over your styles. The helps to avoid repetition and keeps your codebase clean (DRY principle)</answer-component>
+  <question-component>Unmodified sass code can be used in web browsers. True of false?</question-component>
+  <answer-component>False! Sass always needs to be compiled to css to be run in the browser.</answer-component>
+  <question-component>What is a mixin in sass?</question-component>
+  <answer-component>A mixin is a reusable block of sass that inserts rules into the place where you use it.</answer-component>
```

Comment:
Maybe adding an example for a mixin and a function?

Reply:
I think an onboarder should be able to show some code like this somewhere. Therefore it doesn't make sense to me.